documentation/tutorials/prerequisites.md

```diff
@@ -1,6 +1,6 @@
 # Prerequisites
 
-After completing these set up steps, we'll be ready to start developing with the UMA system.
+After completing these set up steps, we'll be ready to start developing with the UMA system locally. 
```

Comment:
I'd actually make it clear that you can do this either locally or on public networks.